src/app/pages/products-list/card/card.component.ts

```diff
@@ -0,0 +1,15 @@
+import {Component} from '@angular/core';
+import {productMock} from 'src/app/shared/products/product.mock';
+
+@Component({
+    selector: 'app-card',
+    templateUrl: './card.component.html',
+    styleUrls: ['./card.component.css'],
+})
+export class CardComponent {
+    product = productMock;
+    cover = this.product.images[0];
+
+    // eslint-disable-next-line no-console
+    buyNow = () => console.log('Buy now');
```

Comment:
Почему стрелочная функция а не классический метод?

Reply:
В данном случае вкусовщина пока не потребуется привязка.

Reply:
Ну и всегда один this даже если вызовем функцию из обьекта(свойства класса)

Reply:
Вкусовщина вкусовщиной, но стоит помнить про консистентность) Если это обычный метод компонента используемый в шаблоне, то я рекомендую его всегда делать самым обычным методом
```suggestion
    buyNow() {
        // eslint-disable-next-line no-console
        console.log('Buy now');
    }
```